static/js/github-create-branch-options.js

```diff
@@ -45,11 +45,9 @@ $(document).ready(() => {
 });
 
 const createUrlForGH = (uuid, multiGHInstance) => {
-	const jiraHost = $("#jiraHost").val();
-	const jwt = $("#jwt").val();
   return uuid ?
-    `github/${uuid}/create-branch?issueKey=${issueKey}&issueSummary=${issueSummary}&jiraHost=${jiraHost}&jwt=${jwt}&ghRedirect=to&multiGHInstance=${multiGHInstance}` :
-    `github/create-branch?issueKey=${issueKey}&issueSummary=${issueSummary}&jiraHost=${jiraHost}&jwt=${jwt}&multiGHInstance=${multiGHInstance}`;
+    `github/${uuid}/create-branch?issueKey=${issueKey}&issueSummary=${issueSummary}&ghRedirect=to&multiGHInstance=${multiGHInstance}` :
```

Comment:
im not sure if JWT still needs to be passed through here, will have a check

Reply:
nope, looks like used locals values now, should be ok

Reply:
I don't think so: once JWT is verified it is put to session and used from there after that https://github.com/atlassian/github-for-jira/blob/e698f8241b16668bd2ec659737061a4ce1b552fd/src/middleware/jira-symmetric-jwt-middleware.ts#L45-L45